_sample_code/osgi_codings_tasks/bundles/org.openhab.training.electricity.tv/src/main/java/org/openhab/training/electricity/tv/TVActivator.java

```diff
@@ -49,7 +49,7 @@ public void start(BundleContext context) throws Exception {
         tracker.open();
 
         // The TV is starting to consume power from the providers
-        tracker.getTV().start();
+        tracker.getTV().startConsuming();
```

Comment:
As we discussed - startConsuming method is already part of the interface and this is the last place it was not fixed. And it is also adapted in the tasks page in the tasks descriptions.